.github/workflows/nightly.yml

```diff
@@ -23,11 +23,10 @@ jobs:
           fetch-depth: 0
 
       - name: Set up JDK ${{ matrix.JABBA_JDK }}
-        uses: olafurpg/setup-scala@v10
-        env:
-          JABBA_INDEX: 'https://github.com/typelevel/jdk-index/raw/main/index.json'
```

Comment:
So I don't know why we were using a custom index, I assume its because at some point the JDK lagged behind but I don't think this is an issue anymore?

Reply:
I don't know or cannot remember, in any case, it shouldn't matter.